### ISSUE TITLE
[java] InvalidLogMessageFormat false-positive for a lambda argument

### ISSUE THREAD

**Issue by boris-petrov** [pmd/pmd#2255]
**Affects PMD Version:** 6.21.0

**Rule:** [InvalidLogMessageFormat](https://pmd.github.io/latest/pmd_rules_java_errorprone.html#invalidlogmessageformat)

**Description:**

This didn't happen on `6.20.0`.

**Code Sample demonstrating the issue:**

```java
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

class InvalidLogMessageFormatTest {
	private static final Logger LOGGER = LoggerFactory.getLogger(InvalidLogMessageFormatTest.class);

	private InvalidLogMessageFormatTest() {
	}

	public static void main(String[] args) {
		foo(exception -> LOGGER.error("Foo", exception));
	}

	private static void foo(Consumer<Throwable> consumer) {
		consumer.accept(new IllegalArgumentException());
	}
}
```

**Comment by adangel** [comment]
Probably related to #2196
